adoc/chapters/programming_interface.adoc

```diff
@@ -19889,6 +19889,35 @@ SYCL provides a number of functions that expose functionality tied to groups of
 work-items (such as <<group-barrier,group barriers>> and collective operations).
 These group functions act as _synchronization points_ and must be encountered in
 converged <<control-flow>> by all work-items in the group.
+
+The behavior of every group function is as follows:
+
+  * Each work-item in the group arrives at the synchronization point associated
+    with the group function, then blocks until any operation(s) specified by the
+    group function have completed.
+
+  * Once all work-items in the group have arrived, an implementation-defined
```

Comment:
```suggestion
  * Once all work-items in the group have arrived, an unspecified
```

Kind of a nit, but I don't think we need implementations to document this subset of work-items.

Reply:
Good catch!